### PR TITLE
[ADP-3239] fix benchmarks on runtime node 8.7.2 (memory + restoration)

### PR DESCRIPTION
**Pull request by paolino**
- [x] Fix memory benchmark
- [x] Fix restoration benchmark

ADP-3239

### CODE REVIEW

File: configs/cardano/mainnet/download.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+curl https://book.play.dev.cardano.org/environments/mainnet/topology.json \
+    > topology.json
+curl https://book.play.dev.cardano.org/environments/mainnet/byron-genesis.json \
+    > byron-genesis.json
+curl https://book.play.dev.cardano.org/environments/mainnet/shelley-genesis.json \
+    > shelley-genesis.json
+curl https://book.play.dev.cardano.org/environments/mainnet/alonzo-genesis.json \
+    > alonzo-genesis.json
+curl https://book.play.dev.cardano.org/environments/mainnet/conway-genesis.json \
+    > conway-genesis.json
```
Comment (HeinrichApfelmus) [inline configs/cardano/mainnet/download.sh]:
```suggestion
    > conway-genesis.json

```

Newline at end of file.

File: configs/cardano/mainnet/download.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
+
+set -euo pipefail
+
+curl https://book.play.dev.cardano.org/environments/mainnet/config.json \
+    > config.json
+curl https://book.play.dev.cardano.org/environments/mainnet/db-sync-config.json \
+    > db-sync-config.json
+curl https://book.play.dev.cardano.org/environments/mainnet/genesis.json \
+    > genesis.json
```
Comment (HeinrichApfelmus) [inline configs/cardano/mainnet/download.sh:10]:
```suggestion
```

That file does not seem to exist? 🤔

File: configs/cardano/mainnet/genesis.json
Comment (HeinrichApfelmus) [inline configs/cardano/mainnet/genesis.json:1]: 4️⃣ 0️⃣ 4️⃣ ⛔

File: configs/cardano/mainnet/download.sh
Comment (HeinrichApfelmus) [inline configs/cardano/mainnet/download.sh:1]: Could you add a `.gitattributes` file, as I did in the `e2e` directory?

https://github.com/cardano-foundation/cardano-wallet/blob/f7cbb90177fe924224834185c3dc24fffd9de8d2/test/e2e/.gitattributes#L1-L9

It turns out that the hashes of the `.json` files are sensitive to line endings.

File: .gitignore [hunk 3]
```diff
@@ -73,3 +73,6 @@ bench-restore
 restore-time.txt
 restore.log
 test-output
+
+## Benchmarks compressed data
```
Comment (HeinrichApfelmus) [inline .gitignore]:
```suggestion
## Benchmarks uncompressed data
```

The `.tgz` file is part of the git repository, right — it's just the uncompressed folder that we exclude here?

**Review by HeinrichApfelmus — APPROVED** [review]
Thank you! 😊 A couple of comments to address before merging, though.

File: .gitignore (hunk 3 above)
Comment (paolino) [reply .gitignore]: yeah, I kind of gave it a different meaning, data that are compressed in the benchmark, but ok, your interpretation is more natural

**Pull request merged**